From: Jan Engelhardt <jengelh@inai.de>
Date: 2015-03-22 15:21:46.000000000 +0100

gnuplot's wxt_gui.cpp (*not* wxWidgets) uses XInitThreads, but
the program does not link in X11 and runs into a build error
otherwise.
---
 gnuplot-5.2.0/src/Makefile.am |    1 +
 1 file changed, 1 insertion(+)

--- gnuplot-5.2.0/src/Makefile.am
+++ gnuplot-5.2.0/src/Makefile.am	2017-09-27 10:06:44.716255786 +0000
@@ -101,6 +101,7 @@ $(EXTRA_gnuplot_SOURCES)
 
 if BUILD_WXWIDGETS
 gnuplot_SOURCES += wxterminal/wxt_gui.cpp
+gnuplot_LDADD += -lX11
 endif
 
 if BUILD_GPCAIRO
